tests/fork/SingleTokenConverter.ts

```diff
@@ -110,6 +144,12 @@ forking(35936683, () => {
           await protocolShareReserve.releaseFunds(CORE_POOL, [BTCB]);
         });
 
+        it("Validates successfull execution of releaseFunds", async () => {
+          for await (const asset of ASSETS) {
```

Comment:
I don't think we need a `for await` here. A simple `for` should be enough

Reply:
fixed - https://github.com/VenusProtocol/protocol-reserve/pull/71/commits/1819985d57eae60fb1e5d6af745e81a1a6191032